doc/static-badges.md

```diff
@@ -0,0 +1,12 @@
+# Static Badges
+
+It is possible to use shields.io to make a wide variety of badges displaying static text and/or logos. For example:
+
+- ![any text you like](https://img.shields.io/badge/any%20text-you%20like-blue) - https://img.shields.io/badge/any%20text-you%20like-blue
+- ![just the messgae](https://img.shields.io/badge/just%20the%20message-8A2BE2) - https://img.shields.io/badge/just%20the%20message-8A2BE2
+- ![for the badge style](https://img.shields.io/badge/for%20the%20badge%20style-20B2AA?style=for-the-badge) - https://img.shields.io/badge/for%20the%20badge%20style-20B2AA?style=for-the-badge
+- ![with a logo](https://img.shields.io/badge/with%20a%20logo-grey?style=for-the-badge&logo=javascript) - https://img.shields.io/badge/with%20a%20logo-grey?style=for-the-badge&logo=javascript
+
```

Comment:
Very good idea to give these badge examples here.
It might be helpful to add also some more badge examples (for now in this file, but also on the re-vamped version of the Shields website):

- how the logo can be added (in different colours), 
- how to add specific URLs from which data can be fetched, and how can the dynamic badges be used (examples related to `query=<[$.DATA.SUBDATA](https://jsonpath.com/)>`).


We can also link this here: https://shields.io/endpoint
`Create badges from your own JSON endpoint.` 
This option is powerful, and I would highlight it, for example:


![Screenshot_4](https://user-images.githubusercontent.com/31891675/205165296-79086f2d-2e71-4536-b643-1d57bc961bb8.jpg)
https://shields.redsparr0w.com/2473/monday 
```js
// 20221201214359
// https://shields.redsparr0w.com/2473/monday

{
  "schemaVersion": 1,
  "label": "is it monday",
  "message": "no",
  "color": "orange"
}
```


Maybe this option would already meet the need of the issue's author :)

Sidenote: Our preview badge on https://shields.io/endpoint doesn't work.
![Screenshot_3](https://user-images.githubusercontent.com/31891675/205165366-095cdd52-edd6-4f5c-bb1e-2fe9ffb2bfb3.jpg)

Reply:
Logos and `logoColor` are covered in https://github.com/badges/shields/blob/master/doc/logos.md which is linked in this doc, so I think that is fine.

The other stuff you've mentioned here is all about dynamic badges, rather than static. I agree they're not as well documented as they could be, but I think the main thing I'm trying to head off here is people raising issues about static badges. Basically since everyone started doing this

![Screenshot at 2022-12-04 12-03-12](https://user-images.githubusercontent.com/6025893/205490260-c6518bcb-51f0-41d9-83e1-521478302871.png)

sort of thing on their profile READMEs, we've had an uptick in these kind of requests so I want to be a bit clearer about it.

On dynamic badges, I'm inclined not to shave this yak any further in this specific PR.

> Sidenote: Our preview badge on https://shields.io/endpoint doesn't work.

The preview for the endpoint badge works, but you need to give it an endpoint to preview.. I think there's a bit of an annoying historical issue here where we can't serve the 'some parameters missing' error when `?url=` is not specified (like with all the other badge routes) because the base URL `/endpoint` doubles up as the docs. We made some bad choices which bit us later. I think the relevant history is:
https://github.com/badges/shields/issues/5014
https://github.com/badges/shields/pull/5137

Reply:
Ok, thanks for the clarification.

You're right, my comments were about dynamic badges, and this PR's goal is to tackle static ones.

Abandoning the subject 🏃 ;)